Logger/FileLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+	    public FileLogger()
+	    {
+	    }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            using (StreamWriter sw = File.AppendText(FilePath))
+            {
+                sw.WriteLine(DateTime.Now + " " + ClassName + " " + logLevel + " " + message);
```

Comment:
String interpolation would make this cleaner.
```suggestion
                sw.WriteLine($"{DateTime.Now} {ClassName} {logLevel} {message}");
```